build.sbt

```diff
@@ -292,6 +292,24 @@ lazy val scioBigtable: Project = Project(
   scioTest % "it"
 ).configs(IntegrationTest)
 
+lazy val scioElsticsearch: Project = Project(
+  "scio-elasticsearch",
+  file("scio-elasticsearch")
+).settings(
+  commonSettings ++ itSettings,
+  description := "Scio add-on for Writing to Elasticsearch Cluster",
+  libraryDependencies ++= Seq(
+    "org.apache.beam" % "beam-sdks-java-io-google-cloud-platform" % beamVersion,
+    "com.google.auto.value" % "auto-value" % "1.2",
+    "joda-time" % "joda-time" % jodaTimeVersion,
+    "org.elasticsearch" % "elasticsearch" % "2.1.0",
```

Comment:
@regadas Do you suggest we support 5.X instead of 2.X? @ravwojdyla mentioned that we should sync with version of Elasticsearch that would be available if we were to setup cluster in GCP. Any thoughts?